projecttacoma/fqm-execution
# Remove EXM130 variants from test fixtures

# Summary
Replaced all EXM130-based fixtures with simple fixtures and ecqm-bundler creations

## New behavior
Test files now pull hand-created fixtures and ecqm-bundler generated bundles instead of overly-complicated EXM130 variants

## Code changes
 - updates to fixtures files and the tests that used to use them

# Testing guidance
 - `npm run check` ensure all unit tests pass as normal
 - Ensure all EXM130-based fixtures are removed
 - Ensure created fixtures are appropriate replacements in all cases
